sycl/doc/extensions/PropertyList/SYCL_EXT_ONEAPI_property_list.asciidoc

```diff
@@ -33,55 +33,50 @@ Because the interfaces defined by this specification are not final and are subje
 
 === Motivation
 
 `sycl::property_list` enables users to pass additional parameters when constructing objects of the runtime classes, but the design of `sycl::property_list` is such that it is not possible to reliably determine the contents of these passed parameters at compile time.
 
-There are numerous circumstances where parameters are only useful when their presence and contents are known at compile-time; this extension addresses this by introducing a `property_list` that is able to represent compile-time-constant properties in addition to runtime properties.
+There are numerous circumstances where parameters are only useful when their presence and contents are known at compile-time; this extension addresses this by introducing a `properties` class that is able to represent compile-time-constant properties in addition to runtime properties.
 
-The handling of runtime properties in `property_list` departs from that of `sycl::property_list` by having the types of any runtime properties present be known at compile-time. The distinction between runtime properties and compile-time-constant properties is that runtime properties store values that are set at runtime.
+The handling of runtime properties in `properties` departs from that of `sycl::property_list` by having the types of any runtime properties present be known at compile-time. The distinction between runtime properties and compile-time-constant properties is that runtime properties store values that are set at runtime.
 
 === Examples
 
 [NOTE]
 ====
-Due to type deduction, users should usually not need to specify the template parameters of `property_list` directly. In cases it is needed `property_list_t` should be used.
+Due to type deduction, users should usually not need to specify the template parameters of `properties` directly. In cases the type is needed, `decltype` should be used.
 ====
 
-A `property_list` that has the compile-time constant property `foo` can be created as follows:
+A property list that has the compile-time constant property `foo` can be created as follows:
 
 ```c++
-property_list P1{foo_v};
+properties P1{foo};
 ```
 
-where `foo_v` is the global variable representing an object of the property value class of the compile-time constant `foo`.
+where `foo` is a global variable representing a property value of the compile-time constant property.
 
-The type of a `property_list` is determined by the set of runtime properties and compile-
-time-constant property values that it contains.  In the following examples, `foo`
-and `bar` are assumed to be compile-time-constant properties, while `baz` is a
+The type of a `properties` object is determined by the set of runtime properties and compile-
+time-constant property values that it contains.  In the following examples, `bar`
+and `baz` are assumed to be compile-time-constant properties, while `foo` is a
```

Comment:
The "assumed to" here strikes me as a little strange.  Who is making the assumption?  Is there a reason this shouldn't just say "`bar` and `baz` are compile-time constant properties"?